scripts/estimate-builds.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+# pylint: disable=invalid-name
+
+import datetime
+from pathlib import Path
+import sys
+
+import croniter
+
+# Add the root of the repo to PYTHONPATH for utils
+ci_root = Path(__file__).resolve().parents[1]
+sys.path.insert(0, str(ci_root))
```

Comment:
Should this go into `get_config_from_generator`? I assume this would be nice to have for any other caller?